ffi/dotnet/Devolutions.IronRdp/src/GatewayConnection.cs

```diff
@@ -0,0 +1,325 @@
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Devolutions.IronRdp;
+
+/// <summary>
+/// Provides methods for connecting to RDP servers through Devolutions Gateway
+/// using the RDCleanPath protocol over WebSocket.
+/// </summary>
+public static class GatewayConnection
```

Comment:
suggestion: Do not hardcode Devolutions Gateway in the type names and documentation. Instead, use RDCleanPath terminology. RDCleanPath is also supported by Cloudflare, so we can potentially integrate with them too.